tests/test_project_management.py

```diff
@@ -113,6 +207,19 @@ def _instrument_service(self, statuses, responses, app=None):
             testutils.MockMultiRequestAdapter(responses, statuses, recorder))
         return recorder
 
+    def _assert_request_is_correct(
+            self, request, expected_method, expected_url, expected_body=None):
+        assert request.method == expected_method
+        assert request.url == expected_url
+        if expected_body is None:
+            assert not request.body
```

Comment:
Nit: Can this be `assert request.body is None`

Reply:
Done.